golang/cosmos/x/lien/keeper/keeper_test.go

```diff
@@ -354,12 +354,6 @@ func TestAccountState(t *testing.T) {
 	if err != nil {
 		t.Fatalf("cannot undelegate: %v", err)
 	}
-	wantState = types.AccountState{
```

Comment:
Don't remove - looks like I forgot to `GetAccountState` and compare. See a few lines above.

Reply:
fixed with a new getstate